cmd/dump/dump.go

```diff
@@ -18,7 +21,15 @@ func main() {
 	pflag.Parse()
 
 	// Hardcoding maxFails to 1 since the purpouse here is just to dump the state once
-	reporter := reporter.NewKubernetesReporter(os.Getenv("ARTIFACTS"), 1)
+	reporter := reporter.NewKubernetesReporter(reporter.KubernetesReporterOptions{
```

Comment:
I think this commit should be squashed into the commit that added the `KubernetesReporterOptions`.